.travis.yml

```diff
@@ -20,8 +20,16 @@ aliases:
   - &run-core-and-ext-tests |
     set -e
     jdk="$(ext/travisci/jdk-from-spec "$PDB_TEST")"
-    if test "$(uname)" = Linux; then jdk_switcher use "$jdk"; fi
     ext/travisci/prep-os-essentials-for "$PDB_TEST"
+    if test "$(uname)" = Linux; then
```

Comment:
Suggest using case and/or OSTYPE (since we know it's bash) here the way we do in some of the other scripts.